04/index.html

```diff
@@ -1,30 +1,47 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 600px)" href="./styles/desktop.css">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
 </head>
+
 <body>
-    
+
     <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+        <button class="hamburger"></button>
+        <ul class="nav__items">
+            <li class="nav__item">
+                <a class="nav__item-link" href="#">start</a>
             </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+            <li class="nav__item">
```

Comment:
Niezbyt ładnie wyglądają te odstępy. Osobiście nadałbym tło dla li bez tych białych wcięć - ale to już zależy od gustu ;)
![image](https://user-images.githubusercontent.com/55095257/170417729-61d8f951-e6ed-4ec8-89db-201e3089a5a1.png)